h1st/model/ml_model.py

```diff
@@ -71,3 +71,6 @@ def base_model(self) -> Any:
     @base_model.setter
     def base_model(self, value):
         setattr(self, "__base_model", value)
+
+    def load_params(self, version: str = None):
+        super().load(version)
```

Comment:
Why does `load_params` change to `load` when calling `super`?

Reply:
@ctn as we have agreed, I have changed "load_params" to "load". 

I am just doing this to make it back ward compatible.

Reply:
@TgithubJ : It should be the same name.

Reply:
@TgithubJ there are two considerations here, they are not mutually exclusive. What is fundamentally correct, and backward compatibility. So the way to make it correct is to propagate this `load_params` as far up the class hierarchy as possible. Then, for backward compatibility, you can maintain the `load` as a synonym *only* where it existed.

PS: Do you once again see the pattern of unnecessary, self-imposed constraint?

Reply:
I got it. Let me fix it.

Reply:
I got it. @phamhoangtuan thanks for fixing this.